mobile/status.go

```diff
@@ -1782,10 +1782,12 @@ func getConnectionStringForBeingBootstrapped(configJSON string) string {
 		return makeJSONResponse(err)
 	}
 
-	err = statusBackend.Logout()
-	if err != nil {
-		return makeJSONResponse(err)
-	}
+	// FIXME: Logout doesn't seem to be necessary here and results in error
+	// in a flow where the device is not logged in
+	// err = statusBackend.Logout()
+	// if err != nil {
+	// 	return makeJSONResponse(err)
+	// }
```

Comment:
Logging out after local pairing (being bootstraped) was very important, some thing didn't work without it initially. 
- https://github.com/status-im/status-go/issues/3399

I guess we never properly tested `getConnectionStringForBeingBootstrapped` flow, because it's not used in desktop/mobile. But it should work, and the API is built in assuming that there will be a logout in the end. 

Even if it works as you say now, I'd better keep the API consistent and keep the `Logout`.

What's exactly the `Logout` error you're getting?